src/components/ViewArticle.jsx

```diff
@@ -26,6 +26,10 @@ export default function ViewArticle(props) {
         })
     }, [])
 
+    function handleVote() {
+        voteOnArticle(currentArticle.article_id)
```

Comment:
consider optimistically rendering the votes here: https://notes.northcoders.com/courses/js-front-end/ui-behaviours-and-optimistic-rendering#optimistic-rendering